src/GovUk.Education.ExploreEducationStatistics.Data.Processor/ImportRecoveryHandler.cs

```diff
@@ -86,31 +86,21 @@ public static void CheckIncompleteImports(string connectionString)
 
         private static TableQuery<DatafileImport> BuildQuery()
         {
-            var combineFilters = TableQuery.CombineFilters(
-                TableQuery.GenerateFilterCondition("Status",
-                    QueryComparisons.Equal, IStatus.QUEUED.ToString())
-                , TableOperators.Or,
-                TableQuery.GenerateFilterCondition("Status",
-                    QueryComparisons.Equal, IStatus.RUNNING_PHASE_1.ToString()));
-
-            combineFilters = TableQuery.CombineFilters(
-                combineFilters,
-                TableOperators.Or,
-                TableQuery.GenerateFilterCondition("Status",
-                    QueryComparisons.Equal, IStatus.RUNNING_PHASE_2.ToString()));
-
-            combineFilters = TableQuery.CombineFilters(
-                combineFilters,
-                TableOperators.Or,
-                TableQuery.GenerateFilterCondition("Status",
-                    QueryComparisons.Equal, IStatus.PROCESSING_ARCHIVE_FILE.ToString()));
-
-            return new TableQuery<DatafileImport>()
-                .Where(TableQuery.CombineFilters(
-                    combineFilters,
-                    TableOperators.Or,
-                    TableQuery.GenerateFilterCondition("Status",
-                    QueryComparisons.Equal, IStatus.RUNNING_PHASE_3.ToString())));
+            var combineFilters = TableQuery.GenerateFilterCondition("Status",
+                    QueryComparisons.Equal, IStatus.QUEUED.ToString());
+
+            IStatus[] statuses = {
+                IStatus.PROCESSING_ARCHIVE_FILE,
+                IStatus.STAGE_1,
+                IStatus.STAGE_2,
+                IStatus.STAGE_3,
+                IStatus.STAGE_4,
+                IStatus.STAGE_5
+            };
+
+            combineFilters = statuses.Aggregate(combineFilters, (current, status) => TableQuery.CombineFilters(current, TableOperators.Or, TableQuery.GenerateFilterCondition("Status", QueryComparisons.Equal, status.ToString())));
```

Comment:
This line is too long and isn't really nice to read (especially compared with what it used to be). Would format it across multiple lines

Reply:
Should be on new lines now